agents/locobot/teleop.py

```diff
@@ -182,9 +222,9 @@ def test_command(sid, commands, yaw_velocity):
             "depthMin": serialized_image["depth_min"],
         })
 
-        pcd = mover.get_current_pcd(in_global=True)
-        points = pcd[0]
-        colors = pcd[1]/255.
+
+        points, colors = rgb_depth.ptcloud.reshape(-1, 3), rgb_depth.rgb.reshape(-1, 3)
+        colors = colors / 255.
```

Comment:
is this normalization necessary here? Was thinking when `reshape` creates a new view versus a copy, and changing the values (by normalization) would create a new copy, right? Do we need that extra overhead?

Reply:
reshape is like a `maybe_copy`. It creates a copy only when a view is not possible.
Here, it's creating views, so colors = colors / 255. is needed